src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohortEngine.java

```diff
@@ -365,7 +365,8 @@ private void processSampleRecordsForLocation(final long location, final Iterable
                     currentPositionHasVariant = true;
                     break;
                 case "0":   // Non Variant Block with GQ < 10
-                    unmergedCalls.add(createRefSiteVariantContextWithGQ(sampleName, contig, currentPosition, refAllele, 0));
+                    // Reference calls with GQ 0 should be rendered as no-call (#271)
```

Comment:
this will functionally include reference calls with GQ >0 but <10, right? that's intended?

Reply:
Right -- post reblocking there are only GQs in increments of 10 (ie 0, 10, 20...). So we are changing the GQ 0s to no calls here, even if that was a GQ2 pre reblocking

Reply:
There is no separate GQ here, for reference calls that is encoded in the state (pet)